src/Microsoft.Data/PrimitiveColumn.cs

```diff
@@ -46,6 +44,25 @@ public PrimitiveColumn(string name, ReadOnlyMemory<byte> buffer, ReadOnlyMemory<
             _columnContainer = new PrimitiveColumnContainer<T>(buffer, nullBitMap, length, nullCount);
         }
 
+        /// <summary>
+        /// Returns an enumerable of immutable memory buffers(and its bit maps) representing the underlying values
+        /// </summary>
+        /// <remarks>Null values are encoded in the returned ReadOnlyMemory<byte> in the Apache Arrow format</remarks>
+        /// <returns>IEnumerable<Tuple<ReadOnlyMemory<typeparamref name="T"/>, ReadOnlyMemory<byte>>></returns>
+        public IEnumerable<Tuple<ReadOnlyMemory<T>, ReadOnlyMemory<byte>>> GetReadOnlyBuffers()
```

Comment:
I wonder if this wouldn't be better as 2 separate APIs... I typically shy away from using Tuple's in an API, because you can't add more members to a Tuple if you need to later.

Reply:
Ah cool. You got to it before I could raise the question. The only reason I chose this was to make the association that this ReadOnlyMemory<T> has it's null values encoded in the associated ReadOnlyMemory<byte>. I'm fresh off of API review though, so, if we added 2 APIs, would users make the association that each ReadOnlyMemory<T> is associated with a ReadOnlyMemory<byte> for the null values? I'm thinking making this explicit in the documentation would be good enough?